addons/hr/__manifest__.py

```diff
@@ -57,6 +57,12 @@
         'web.assets_backend': [
             'hr/static/src/**/*',
         ],
+        'web.assets_frontend': [
+            'hr/static/src/core/common/**/*',
+        ],
```

Comment:
We usually avoid the front-end bundle as it is not lazy loaded.
If we don't have the store in the bundle we shouldn't have any other file either.
Which issue is it solving? Maybe it should be in the lazy portal bundle, and/or in the livechat bundle?